Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.cs

```diff
@@ -431,9 +444,12 @@ private void UpdateCurrentTextEdit(PretokenStringContainer edit)
             Text = edit.Text; // Update our text property.
         }
 
-        private async Task RemoveToken(TokenizingTextBoxItem item)
+        private async Task<bool> RemoveToken(TokenizingTextBoxItem item, object data = null)
```

Comment:
Since this is private, I don't care as much, but I would also change it to RemoveTokenAsync.